opa-manager-operator/testing-yamls/rego.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
```

Comment:
Where is this file used?

Reply:
these are example files, maybe I should not have commited. Will remove from next commit